nixos/modules/services/x11/picom.nix

```
@@ -61,6 +61,8 @@ in {
       '';
     };
 
+    package = mkPackageOption pkgs "picom" { };
```

```suggestion
    package = mkPackageOptionMD pkgs "picom" { };
```
We're in the process of moving docs to MD in nixpkgs.

---

As of d36f950d40536, `mkPackageOptionMD` is an alias for `mkPackageOption` :)